File: Implementation/Algorithms/cpp/Greedy/Ex02.cpp
/*
 *
 * Problem: https://codeforces.com/problemset/problem/401/C
 *
 */

#include <bits/stdc++.h>

using namespace std;

int main()
{
    int a, b, c = 0, d = 0;
    cin >> a;

    while (a--)
    {
        for (int i = 0; i < 3; ++i)
        {
            cin >> b;
            c += b;
        }

        if (c > 1) ++d;

        c = 0;
    }

    cout << d << "\n";
    
    return 0;
}

File: Implementation/Algorithms/python/Greedy/Ex02.py
 # Problem: https://codeforces.com/problemset/problem/401/C

a = input().split()
c = 0
d = 0

while a > 0:
    b = [int(i) for i in input().split()]
    for i in b:
        c += b[i]


    if c > 1:
        d += 1

    a -= 1

print(d)
